beacon-chain/rpc/eth/shared/errors.go

```diff
@@ -43,3 +47,15 @@ type IndexedVerificationFailure struct {
 	Index   int    `json:"index"`
 	Message string `json:"message"`
 }
+
+// WriteStateFetchError writes an appropriate error based on the supplied argument.
+// The argument error should be a result of fetching state.
+func WriteStateFetchError(w http.ResponseWriter, err error) {
```

Comment:
I wrote this at the same time @saolyn implemented `HandleStateFetchError`. I decided to keep this function because the `shared` package is used by HTTP. The `helpers` one will probably be removed once we don't have grpc endpoints anymore.